doc/src/index.md

```diff
@@ -1,4 +1,15 @@
-# Julia Documentation
+# Julia v0.6 Documentation
+
+Welcome to the documentation for Julia 0.6!
+
+### What's new in Julia 0.6
+
+The [Release notes](NEWS.md) contains a list of new features, changes and deprecations.
```

Comment:
"Release" -> "release", and "contains" -> "contain"? Perhaps simplify to "The release notes list new features, changes, and deprecations."?